tck/api/src/main/java/org/eclipse/microprofile/metrics/tck/metrics/HistogramTest.java

```diff
@@ -140,34 +141,72 @@ public void testSnapshotValues() throws Exception {
                 SAMPLE_LONG_DATA, histogramLong.getSnapshot().getValues());
     }
 
+    @Test
+    public void testSnapshotPercentileValuesPresent() throws Exception {
```

Comment:
not sure if we really need to test this?

Reply:
seems reasonable for this release, given those are the expected percentiles